src/Qir/Tests/QIR-dynamic/qsharp/qir-test-assert.qs

```diff
@@ -1,15 +1,235 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// https://docs.microsoft.com/en-us/azure/quantum/concepts-the-qubit#single-qubit-operations    Qubit and Operations
+// https://docs.microsoft.com/en-us/azure/quantum/user-guide/libraries/standard/prelude         Operation Details, Multi-Qubit Operations
+
 namespace Microsoft.Quantum.Testing.QIR  {
 
     open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    // |0> -> |1>
+    @EntryPoint()
+    operation AsrtMeasAlloc1OKTest() : Unit {
+        use qubit = Qubit();    // |0>
+
+        AssertMeasurement(           [PauliZ], [qubit], Zero,      "0: Newly allocated qubit must be in the |0> state.");
+        AssertMeasurementProbability([PauliZ], [qubit], Zero, 1.0, "1: Newly allocated qubit must be in the |0> state.", 1e-10);
+
+        X(qubit);   // |0> -> |1>
+
+        AssertMeasurement(           [PauliZ], [qubit], One,      "2: Newly allocated qubit after X() must be in the |1> state.");
+        AssertMeasurementProbability([PauliZ], [qubit], One, 1.0, "3: Newly allocated qubit after X() must be in the |1> state.", 1e-10);
+    }
+
+    // Fail: Expecting |1>, should have expected |0>
+    @EntryPoint()
+    operation AsrtMeasAlloc1ExcTest() : Unit {
+        use qubit = Qubit();    // |0>
+
+        // Must fail:
+        AssertMeasurement([PauliZ], [qubit], One, "Newly allocated qubit must be in the |0> state.");
+    }
+
+    // Fail: Expecting |1>, should have expected |0>
+    @EntryPoint()
+    operation AsrtMeasProbAlloc1ExcTest() : Unit {
+        use qubit = Qubit();    // |0>
+
+        // Must fail:
+        AssertMeasurementProbability([PauliZ], [qubit], One, 1.0, "Newly allocated qubit must be in the |0> state.", 1e-10);
+    }
+
+    @EntryPoint()
+    operation AsrtMeasProbAlloc1HalfProbTest() : Unit {     //!
```

Comment:
What does the `//!` indicate here?

Reply:
Sorry, that was just a temporary bookmark for myself. Removed.